website/payments/models.py

```diff
@@ -480,3 +481,78 @@ def __str__(self):
 
     class Meta:
         ordering = ("created_at",)
+
+
+class PaymentRequest(Payable):
+    requester = models.OneToOneField(Member, on_delete=models.CASCADE)
+
+    request_timestamp = models.DateTimeField(
+        verbose_name=_("request timestamp"),
+        blank=False,
+        null=False,
+    )
+
+    payer = models.OneToOneField(PaymentUser, on_delete=models.CASCADE)
+
+    payment_timestamp = models.DateTimeField(
+        verbose_name=_("payment timestamp"),
+        blank=True,
+        null=True,
+    )
+
+    amount = models.FloatField(
+        verbose_name=_("payment amount"),
+        blank=False,
+        null=False,
+    )
+
+    topic = models.TextField(
+        max_length=32,
+        verbose_name=_("payment topic"),
+        blank=False,
+        null=False,
+    )
+
+    notes = models.TextField(
+        max_length=256,
+        verbose_name=_("payment notes"),
+        blank=True,
+        null=True,
+    )
+
```

Comment:
There should probably be some field to decide (for bookkeeping purposes) whether the payable 'has to be paid sometime'? I don't know enough about that though.

Reply:
Should we confirm with roel/nick/someone else that knows about bookkeeping what requirements this field should uphold?